examples/nlp/glue_with_BERT.py

```diff
@@ -75,6 +75,7 @@
     eval_iter_callback, eval_epochs_done_callback
 
 from nemo_nlp.data.datasets.utils import processors, output_modes
+from pytorch_transformers import BertConfig
```

Comment:
it might be a good idea to wrap a NeMo module around BertConfig so we don't expose pytorch_transformers in the application script, but i'm not sure given that transformers now support both TF and PT. What do you think @okuchaiev?

Reply:
@chiphuyen I agree that we should avoid "from pytorch_transformers import BertConfig" going forward.
@yzhang123 is there a quick work-around to not do this? If not, I think we can still merge this PR since it is a "bugfix", but we'll have to go back to this eventually

Reply:
@okuchaiev 699f5a4339e76e4c6db1ebba79fec5596414fe07